extensions/stackdriver/stackdriver.cc

```diff
@@ -221,6 +219,17 @@ bool StackdriverRootContext::configure(size_t configuration_size) {
             configuration + ", " + status.message().ToString());
     return false;
   }
+
+  int duration = 0;
```

Comment:
If I remembered correctly, topology report is also control by this ticker. Could you check if there is any side effect on that?

Reply:
As we talked offline.. added a default timer for ticker...